pkgs/development/interpreters/boron/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchgit
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "boron";
+  version = "2.0.8";
+
+  src = fetchgit {
+    url = "https://git.code.sf.net/p/urlan/boron/code";
+    rev = "v${version}";
+    sha256 = "sha256-RpnMs9DqcbK/gLDH5x3/DvxodJb4tR76Yk8SbYtIkV4=";
+  };
+
+  # this is not a standard `configure`
+  configurePhase = ''
+    patchShebangs .
+    ./configure
+  '';
+
+  buildPhase = ''
+    export DESTDIR=$out
+    make -j
+    mkdir -p $out/include/boron
+    cp include/* $out/include/boron
+  '';
+
+  buildInputs = [
+    zlib
+  ];
+
+  postInstall = ''
+    ln -s libboron.so.2 $out/lib/libboron.so
+  '';
```

Comment:
It's better to specify install targets, like `installTargets = "install install-dev";`.

Reply:
Applied in https://github.com/NixOS/nixpkgs/pull/180865/commits/3fbaaa44942c196d937dc451ffcc88ea86e3ec32